pkg/unvanquished_src.dpkdir/ui/options_graphics.rml

```diff
@@ -260,6 +296,29 @@
 				</buttonbar>
 			</panel>
 
+			<tab><translate>Experimental</translate></tab>
+			<panel>
+				<h2><translate>Material system</translate></h2>
+				<row>
+					<h3><translate>Material system</translate></h3>
+					<input cvar="r_materialSystem" type="checkbox" />
+					<p><translate>Use a new system that significantly improves performance. Some drivers may fail with this on (e. g. Mesa radeonsi and AMD proprietary drivers).</translate></p>
+				</row>
+				<row>
+					<h3><translate>GPU frustum culling</translate></h3>
+					<input cvar="r_gpuFrustumCulling" type="checkbox" />
+					<p><translate>Cull surfaces that are out of view.</translate></p>
```

Comment:
Maybe `Use the GPU to cull surfaces that are out of view.` and `Use the GPU to cull hidden surfaces.`?

Reply:
I  could do that, but the options do both say "GPU ... culling".

Reply:
Ah yes, that's fine then.